R/class-spec.R

```diff
@@ -187,7 +187,9 @@ class_inherits <- function(x, what) {
 }
 
 obj_type <- function(x) {
-  if (inherits(x, "S7_object")) {
+  if (identical(x, quote(expr = ))) {
```

Comment:
`rlang::is_missing()` is:

```r
missing(x) || identical(x, quote(expr = ))
```

do we need that `missing(x)` bit?

Reply:
I don't think it's important here.